homeworks/homework3/README.md

```diff
@@ -0,0 +1,4 @@
+# Homework №3
+The task of analyzing tables and their visualization.
+
+[Solution](https://datalore.jetbrains.com/notebook/2mZNiVo8e2qfGRPISuqC6S/zqeDyoZ9GsYZwupCsvLfst/)
```

Comment:
In **1c** (Task 3), I cannot replicate your results, it seems that this is because you use `(data["Survived"] == "Yes")`, and you do not use `Yes`.

Reply:
Moreover, even in the results that you notebook shows, there are actually two people tied for the least money paid. Write the code that will print the top and the bottom and will take into account that several people can be there.

Reply:
The same goes for Task 4, you use the `Yes` keyword (the answer is correct though)

Reply:
In `3D Scatter plot`, your categorical PClass is in random order, please, sort it.

Reply:
The same problem in `Box chart #1`